views/js/qtiCreator/plugins/panel/outcomeEditor.js

```diff
@@ -61,15 +76,23 @@ define([
     function renderListing(item, $outcomeEditorPanel){
 
         var rpVariables = getRpUsedVariables(item);
-        
+
         var outcomesData = _.map(item.outcomes, function(outcome){
             var readonly = (rpVariables.indexOf(outcome.id()) >= 0);
+
+            const externalScored = {
+                none : {label : __("None"), selected : !outcome.attr('externalScored')},
```

Comment:
Inconsistent quotes. Here is double, elsewhere is single. Is it done on purpose?